packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -47,6 +49,32 @@ interface TokensListProps {
 }
 
 const getTokensByTheme = (theme: ThemeVariants, props: TokensListProps): TokenCategory[] => {
+  const font = props.consoleTokens[0].node.tokens.find(ele => ele.categoryName === 'fonts');
+  if (font) {
+    font.info = (
+      <Callout variant="secondary">
```

Comment:
@serifluous is the secondary variant a standard approach for us in callouts like this?

Reply:
Great call out I was just shooting from the hip. I figured secondary was a good info variant

Reply:
Poke for feedback on this @serifluous

Reply:
Let's use the default (no variant) since we haven't used the secondary variant elsewhere.